isis/src/base/objs/Stretch/Stretch.h

```diff
@@ -90,13 +89,10 @@ namespace Isis {
       double p_minimum; //!<By default this value is set to p_lrs
       double p_maximum; //!<By default this value is set to p_hrs
 
-      QString p_type; //! Type of stretch. This is only currently used in the AdvancedStretchTool.
```

Comment:
It seems like in this file you are removing the blob reading/writing capability and you add that to the Blob object, which totally makes sense. I am just curious why you are removing the stretch type property from here?

Reply:
:+1: The stretchType (Binary, Sawtooth, Linear, or Manual) property only really makes sense in the context of the Advanced Stretch Tool, and I'd originally added it to this Stretch class the last time I worked on this "Save stretch" project. The Stretch base class is used to represent a stretch pair of series of stretch pairs, and is used in various places in ISIS besides the context of saving/loading stretches from cubes. I added the CubeStretch class in this PR to specifically deal with the information needed in a saved stretch for a cube, which includes the stretch type and a band number.

Reply:
👍 That all makes sense and is way more clean.